app/src/main/java/com/alexis/myanimecompanion/data/RemoteDataSource.kt

```diff
@@ -0,0 +1,135 @@
+package com.alexis.myanimecompanion.data
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.util.Base64
+import com.alexis.myanimecompanion.data.remote.APIClient
+import com.alexis.myanimecompanion.data.remote.MyAnimeListAPI
+import com.alexis.myanimecompanion.data.remote.models.RemoteToken
+import com.alexis.myanimecompanion.data.remote.models.asAnime
+import com.alexis.myanimecompanion.data.remote.models.asListOfAnime
+import com.alexis.myanimecompanion.domain.Anime
+import retrofit2.HttpException
+import java.security.SecureRandom
+import androidx.security.crypto.EncryptedSharedPreferences
+
+import androidx.security.crypto.MasterKeys
+
+
```

Comment:
Remove extra lines.
Seems it got added in this [commit](https://github.com/AlexisN94/MyAnimeCompanion/pull/1/commits/007187885442bc6559561a9fa6b864fd3e9db8a8)